### DISCUSSION TITLE
Default killswitch is turned off

### DISCUSSION THREAD

**Discussion by DevZenFlow** [bubuntux/nordvpn#141] [Q&A]
Hi, 

The docs say that kill switch is on by default, but when starting the container in debug mode, the logs suggest the switch is off. Is there an ENV variable I can set to enable the switch? 

Logs:

Technology: NordLynx
vpn_1           | Kill Switch: **disabled**
vpn_1           | CyberSec: disabled
vpn_1           | Notify: disabled
vpn_1           | Auto-connect: disabled
vpn_1           | DNS: disabled

**Comment by bubuntux** [comment]
The kill switch is done using iptable rules, is safer since they apply even before you connect to the vpn

**Reply by bubuntux** [reply]
you can take a look at the script https://github.com/bubuntux/nordvpn/blob/master/start_vpn.sh
